cmd/geth/accountcmd.go

```diff
@@ -316,7 +316,7 @@ func accountImport(ctx *cli.Context) error {
 	}
 	key, err := crypto.LoadECDSA(keyfile)
 	if err != nil {
-		utils.Fatalf("keyfile must be given as argument")
```

Comment:
Could you please update it a bit to include the error message? I.e.

```
utils.Fatalf("Failed to load the private key: %v", err)
```

(note, capitalized, and error at the end)